backend/api/appsettings.Local.json

```diff
@@ -2,20 +2,6 @@
   "HealthChecks": {
     "Port": "5000"
   },
-  "HealthChecksUI": {
```

Comment:
this doesn't seem related to the title of the PR, intended?

Reply:
I did include it in the description of the PR.  I am working on the prod pipeline.  While making sure the DB migration occurs during a deploy I also cleaned up some of the code in the related method.  Additionally I've updated the dependencies and removed one that isn't required anymore.